.github/workflows/ci.yml

```diff
@@ -42,8 +42,7 @@ jobs:
     strategy:
       matrix:
         rust:
-          # This is the minimum supported Rust version for "no_std"
-          - 1.36.0
+          - 1.46.0
```

Comment:
Now that this is the same as MSRV with "std" feature, it can be merged with the MSRV job above.

Something like:

```
cargo hack build --rust-version --no-default-features --target thumbv7m-none-eabi
```